src/ctia/lib/utils.clj

```diff
@@ -173,3 +173,21 @@
                           service-fns)))))
           {}
           (partition-all 2 selectors)))
+
+(defn assoc-new-keys
+  "Like assoc, except only associates keys contained in m.
+  Note: for duplicate keys in kv, last wins.
+  
+  (assoc-new-keys m k v ...) is like (into (hash-map k v ...) m)
```

Comment:
Hmmm... this statement is a bit confusing:

```
(into (hash-map :foo 2) {:foo 1}) ; => {:foo 1}
(into (hash-map :foo 1) {:foo 2}) ; => {:foo 2}
(assoc-new-keys {:foo 1} :foo 2) ; => {:foo 1}
```

For data transform functions like that, maybe it's useful to attach some examples. Perhaps even something like this: https://github.com/threatgrid/ctia/blob/master/test/ctia/test_helpers/pagination.clj#L128

Reply:
Thanks, I'm going to ditch this for `into`.

Reply:
It's always the helpers that get feedback isn't it... :)

Reply:
FWIW your second example had the kv and m backwards from the description. The first and third example matched in both pattern and result.

Reply:
> It's always the helpers that get feedback isn't it... :)

Probably because they are small and easily REPLable.

Reply:
Easy bikesheddable! :D